src/compress.c

```diff
@@ -127,6 +127,8 @@ void rsDecompressCreate(RSDecompress *decompress) {
    decompress->source.jpeg.fill_input_buffer = decompressSourceFill;
    decompress->source.jpeg.skip_input_data = decompressSourceSkip;
    decompress->source.jpeg.resync_to_restart = jpeg_resync_to_restart;
+   decompress->source.filled = false;
+   decompress->source.jpeg.bytes_in_buffer = 0;
```

Comment:
Aren't these set by the init and fill callbacks?

Reply:
valgrind complains about source.filled being not initialized, there's an if based on that value, if it's not initialized that'd be very bad. However, this is from back when I had those weird memory corruption bugs. I'll remove it and see if it works.